Plugin: 明日方舟工具箱

**插件名称：**

明日方舟工具箱

**插件功能：**

基于 OneBot 适配器的 NoneBot2 明日方舟小工具箱插件

**PyPI 项目名：**

nonebot-plugin-arktools

**插件 import 包名：**

nonebot_plugin_arktools

**插件项目仓库/主页链接：**

https://github.com/NumberSir/nonebot_plugin_arktools

**标签：**

[{"label":"a:onebot","color":"#000000"},{"label":"arknights","color":"#22bbff"},{"label":"t:game","color":"#db905e"}]

# 📃 商店发布检查结果

> Plugin: 明日方舟工具箱

**⚠️ 在发布检查过程中，我们发现以下问题：**
<pre><code><li>⚠️ 插件加载测试未通过。<details><summary>测试输出</summary>插件 nonebot-plugin-arktools 的信息如下：
    name         : nonebot-plugin-arktools            
     version      : 1.0.0                              
     description  : 基于 OneBot 适配器的 NoneBot2 明日方舟小工具箱插件 
    
    dependencies
     - aiofiles &gt;=0.8.0
     - httpx &gt;=0.23.1
     - lxml &gt;=4.9.2
     - nonebot-adapter-onebot &gt;=2.2.0
     - nonebot-plugin-apscheduler &gt;=0.2.0
     - nonebot-plugin-imageutils &gt;=0.1.14
     - nonebot2 &gt;=2.0.0rc2
     - tortoise-orm &gt;=0.19.3
插件 nonebot_plugin_arktools 加载出错：
02-12 18:30:46 [SUCCESS] nonebot | NoneBot is initializing...
02-12 18:30:46 [INFO] nonebot | Current Env: prod
02-12 18:30:47 [ERROR] nonebot | Failed to import &quot;nonebot_plugin_arktools&quot;
Traceback (most recent call last):
  File &quot;/home/runner/work/nonebot2/nonebot2/plugin_test/runner.py&quot;, line 4, in &lt;module&gt;
    valid = load_plugin(&quot;nonebot_plugin_arktools&quot;)
  File &quot;/home/runner/.cache/pypoetry/virtualenvs/plugin-test-rYmcgvOP-py3.10/lib/python3.10/site-packages/nonebot/plugin/load.py&quot;, line 34, in load_plugin
    return manager.load_plugin(module_path)
&gt; File &quot;/home/runner/.cache/pypoetry/virtualenvs/plugin-test-rYmcgvOP-py3.10/lib/python3.10/site-packages/nonebot/plugin/manager.py&quot;, line 141, in load_plugin
    module = importlib.import_module(name)
  File &quot;/usr/lib/python3.10/importlib/__init__.py&quot;, line 126, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File &quot;&lt;frozen importlib._bootstrap&gt;&quot;, line 1050, in _gcd_import
  File &quot;&lt;frozen importlib._bootstrap&gt;&quot;, line 1027, in _find_and_load
  File &quot;&lt;frozen importlib._bootstrap&gt;&quot;, line 1006, in _find_and_load_unlocked
  File &quot;&lt;frozen importlib._bootstrap&gt;&quot;, line 688, in _load_unlocked
  File &quot;/home/runner/.cache/pypoetry/virtualenvs/plugin-test-rYmcgvOP-py3.10/lib/python3.10/site-packages/nonebot/plugin/manager.py&quot;, line 233, in exec_module
    super().exec_module(module)
  File &quot;&lt;frozen importlib._bootstrap_external&gt;&quot;, line 883, in exec_module
  File &quot;&lt;frozen importlib._bootstrap&gt;&quot;, line 241, in _call_with_frames_removed
  File &quot;/home/runner/.cache/pypoetry/virtualenvs/plugin-test-rYmcgvOP-py3.10/lib/python3.10/site-packages/nonebot_plugin_arktools/__init__.py&quot;, line 1, in &lt;module&gt;
    from .src import *
  File &quot;/home/runner/.cache/pypoetry/virtualenvs/plugin-test-rYmcgvOP-py3.10/lib/python3.10/site-packages/nonebot_plugin_arktools/src/__init__.py&quot;, line 1, in &lt;module&gt;
    from .game_guess_operator import *  # 猜干员
  File &quot;/home/runner/.cache/pypoetry/virtualenvs/plugin-test-rYmcgvOP-py3.10/lib/python3.10/site-packages/nonebot_plugin_arktools/src/game_guess_operator/__init__.py&quot;, line 19, in &lt;module&gt;
    from .data_source import *
  File &quot;/home/runner/.cache/pypoetry/virtualenvs/plugin-test-rYmcgvOP-py3.10/lib/python3.10/site-packages/nonebot_plugin_arktools/src/game_guess_operator/data_source.py&quot;, line 1, in &lt;module&gt;
    from ..core.models_v3 import Character
  File &quot;/home/runner/.cache/pypoetry/virtualenvs/plugin-test-rYmcgvOP-py3.10/lib/python3.10/site-packages/nonebot_plugin_arktools/src/core/__init__.py&quot;, line 1, in &lt;module&gt;
    from .models_v3 import *
  File &quot;/home/runner/.cache/pypoetry/virtualenvs/plugin-test-rYmcgvOP-py3.10/lib/python3.10/site-packages/nonebot_plugin_arktools/src/core/models_v3.py&quot;, line 1508
    idx: [
         
SyntaxError: asynchronous comprehension outside of an asynchronous function</details></li></code></pre>
<details><summary>详情</summary><pre><code><li>✅ 标签: a:onebot-#000000, arknights-#22bbff, t:game-#db905e。</li><li>✅ 项目 <a href="https://github.com/NumberSir/nonebot_plugin_arktools">主页</a> 返回状态码 200。</li><li>✅ 包 <a href="https://pypi.org/project/nonebot-plugin-arktools/">nonebot-plugin-arktools</a> 已发布至 PyPI。</li></code></pre></details>

---

💡 如需修改信息，请直接修改 issue，机器人会自动更新检查结果。
💡 当插件加载测试失败时，请发布新版本后在当前页面下评论任意内容以触发测试。

💪 Powered by [NoneBot2 Publish Bot](https://github.com/nonebot/nonebot2-publish-bot)
<!-- PUBLISH_BOT -->

--- a/website/static/plugins.json
+++ b/website/static/plugins.json
@@ -4321,5 +4321,28 @@
       }
     ],
     "is_official": false
+  },
+  {
+    "module_name": "nonebot_plugin_arktools",
+    "project_link": "nonebot-plugin-arktools",
+    "name": "明日方舟工具箱",
+    "desc": "基于 OneBot 适配器的 NoneBot2 明日方舟小工具箱插件",
+    "author": "NumberSir",
+    "homepage": "https://github.com/NumberSir/nonebot_plugin_arktools",
+    "tags": [
+      {
+        "label": "a:onebot",
+        "color": "#000000"
+      },
+      {
+        "label": "arknights",
+        "color": "#22bbff"
+      },
+      {
+        "label": "t:game",
+        "color": "#db905e"
+      }
+    ],
+    "is_official": false
   }
 ]

